.github/workflows/ci.yaml

```diff
@@ -40,44 +40,44 @@ jobs:
         node-version: [14.x]
     runs-on: ${{ matrix.platform }}
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Run unit tests
-      run: |
-        make unit-test-ts
-    - name: Check licenses
-      run: |
-        cd functions/ts
-        make check-licenses
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Run unit tests
+        run: |
+          make unit-test-ts
+      - name: Check licenses
+        run: |
+          cd functions/ts
+          make check-licenses
 
   go-unit-test-ci:
     runs-on: ubuntu-latest
     env:
       GOPATH: /home/runner/work/kpt-functions-catalog/functions/go
       GO111MODULE: on
     steps:
-    - name: Set up Go 1.17
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.17
-      id: go
-    - name: Check out code into GOPATH
-      uses: actions/checkout@v1
-      with:
-        path: go/src/github.com/${{ github.repository }}
-    - name: Run unit tests
-      run: |
-        make unit-test-go
-    - name: Check licenses
-      run: |
-        cd functions/go
-        make check-licenses
+      - name: Set up Go 1.17
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+        id: go
+      - name: Check out code into GOPATH
+        uses: actions/checkout@v1
+        with:
+          path: go/src/github.com/${{ github.repository }}
+      - name: Run unit tests
+        run: |
+          make unit-test-go
+      - name: Check licenses
+        run: |
+          cd functions/go
+          make check-licenses
 
   e2e-ci:
-    needs: [ts-unit-test-ci, go-unit-test-ci]
```

Comment:
It's not necessary to block the e2e tests.

Reply:
Does this mean, this job will start in parallel with the `ts-unit-test-ci` and `go-unit-test-ci` ?

Reply:
Yes. Do you have any concerns?

Reply:
No concerns. Just happy :)